{{ cookiecutter.repo_name }}/docs/user_guide/loading_environment_variables.md

```diff
@@ -73,5 +81,7 @@ source_env ".secrets"
 This ensures [`direnv`][direnv] loads the `.secrets` file using `.envrc` without
```

Comment:
Is this secrets documentation still correct?

Reply:
Updated